src/Cart.js

```diff
@@ -19,15 +50,28 @@ class Cart extends Component {
                     <h1>Here are all the products in your cart!</h1>
                     <ul>
                         {lineItems.map(p=>{
+                            const total = p.quantity * p.price;
                             return (
+                                <div>
```

Comment:
This page is getting an error due to the lack of a key prop on the div. That said, you probably don't actually need the div. The li wraps all other elements, and has no adjacent elements. I would recommend removing the div.